lib/puppet/network/formats.rb

```diff
@@ -151,7 +151,7 @@ def render(datum)
     end
 
     # ...or pretty-print the inspect outcome.
-    return json.render(datum)
+    return JSON.pretty_generate(datum)
```

Comment:
I opened a PR https://github.com/puppetlabs/puppet/pull/3672

@dalen sorry that we led you down a dark alley way to switch from PSON.pretty_generate to JSON.pretty_generate, only to find out it doesn't work.